victoriensukarieh.Calculator/ConsoleCalculator/Models/Operation.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace ConsoleCalculator.Models;
```

Comment:
Please tidy up code formating: This button in the bottom of the code window is very useful for that: 😁
![image](https://user-images.githubusercontent.com/108711476/228714194-e1226102-3f3e-45a3-a05a-95f03f995313.png)